tests/acceptance/test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+import _ from 'lodash';
+import {expect} from 'chai';
+import webdriverio from 'webdriverio';
+import path from 'path';
+
+const BROWSER_OPTIONS = {
+  desiredCapabilities: {
+    browserName: 'PhantomJS'
+  }
+};
+const VIEWPORT_SIZE = {
+  width: 1024,
+  height: 768
+};
+const URL = 'file://' + path.join(__dirname, '..', '..', 'visual-report',
+            'statics', 'index.html');
+
+describe('Mocha Mugshot acceptace tests', function() {
+  let client;
+
+  before(function(done) {
+    client = webdriverio.remote(BROWSER_OPTIONS).init()
+      .setViewportSize(VIEWPORT_SIZE)
+      .url(URL)
+      .then(function() {
+        done();
```

Comment:
Return the promise instead of using callbacks. Mocha knows to wait for a promise to be fulfilled.